### ISSUE TITLE
Bring back chainsail-resources?

### ISSUE THREAD

**Issue by steshaw** [tweag/chainsail#448]
Now that Chainsail is open-source, it makes a lot of sense to bring back https://github.com/tweag/chainsail-resources/ into the main repository.

**Comment by simeoncarstens** [comment]
I agree, there's no reason anymore to have two separate repositories.
